packages/components/Menu/src/MenuTrigger/useMergeRefs.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copied from @fluentui/react-utilities
+ *
+ * Keeping the file local to Menu for now while looking into whether
+ * we can pull in @fluentui/react-utilities package
```

Comment:
Adding to interactive-hooks would add to public API and mean that if we change the source of the functionality, it would be taken back.